decimal.d.ts

```diff
@@ -523,6 +523,38 @@ export declare class Decimal {
      * Configure global settings for a Decimal constructor.
      */
     static set(config: Config): Decimal;
+
+    // The maximum number of significant digits of the result of a calculation or base conversion.
+    // E.g. `Decimal.config({ precision: 20 });`
+    static readonly precision: number;
+
+    // The rounding mode used by default by `toInteger`, `toDecimalPlaces`, `toExponential`,
+    // `toFixed`, `toPrecision` and `toSignificantDigits`.
+    //
+    // E.g.
+    // `Decimal.rounding = 4;`
+    // `Decimal.rounding = Decimal.ROUND_HALF_UP;`
+    static rounding: number;
```

Comment:
Should this also have been `readonly`?

Reply:
It should be not `readonly` to allow the e.g. code `Decimal.rounding = 4;`

Reply:
Yes, but the same could be said for `precision`, `toExpNeg`, `toExpPos` and `LN10`.

I think it is best to have them all __not__ `readonly`, with `readonly` only for the rounding modes, i.e. `ROUND_UP` etc.

Reply:
fixed in 5cd4531

Reply:
Okay, thanks.